app/electron/main/windows.js

```diff
@@ -1,13 +1,18 @@
 import {BrowserWindow, Menu, dialog, ipcMain, webContents} from 'electron';
+import {join} from 'path';
 
 import settings from '../../common/shared/settings';
 import i18n from './i18next';
 import {openFilePath} from './main';
 import {APPIUM_SESSION_EXTENSION, isDev} from './helpers';
 import {rebuildMenus} from './menus';
 
-const mainUrl = `file://${__dirname}/index.html`;
-const splashUrl = `file://${__dirname}/splash.html`;
+const mainPath = isDev
+  ? process.env.ELECTRON_RENDERER_URL
+  : join(__dirname, '../renderer/index.html'); // from 'main' in package.json
```

Comment:
consider using the `join(__dirname, '..', 'renderer', 'index.html')` format as it should properly handle path reparators